app/(landing)/Schedule.tsx

```diff
@@ -17,8 +17,8 @@ export default function Schedule() {
                     <ScheduleItem start="7:00 PM" end="7:30 PM">Coding Trivia Kahoot</ScheduleItem>
                     <ScheduleItem start="7:30 PM" end="8:45 PM">Discord Bots (Python + JS)</ScheduleItem>
                     <ScheduleItem start="8:45 PM" end="10:00 PM">Basic AI/ML</ScheduleItem>
-                    <ScheduleItem start="10:00 PM" end="10:30 AM">Cow Drawing Competition</ScheduleItem>
-                    <ScheduleItem start="11:00 PM" end="11:30 AM">Chess Tournament</ScheduleItem>
+                    <ScheduleItem start="10:00 PM" end="10:30 PM">Cow Drawing Competition</ScheduleItem>
+                    <ScheduleItem start="11:00 PM" end="11:30 PM">Chess Tournament</ScheduleItem>
                 </ScheduleDay>
                 <ScheduleDay name="January 22">
                     <ScheduleItem start="12:00 AM" end="2:00 AM">Movie Night & Snacks</ScheduleItem>
```

Comment:
We should probably decide if we're doing `Title Case` for all of these or not. If so, the first 3 schedule items need to be updated, though personally I'm not a fan of using Title Case for things that aren't names (ie. workshops or activities).